src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.pushAlive(this);
+  }
+
+  static pushAlive(animal) {
+    Animal.alive.push(animal);
+  }
+
+  static removeAnimal(animalDied) {
+    Animal.alive
+    = Animal.alive.filter(animal => animal.name !== animalDied.name);
```

Comment:
that looks weird, better move your filter call on a new line
```suggestion
    Animal.alive = Animal.alive
    	.filter(animal => animal.name !== animalDied.name);
```